apps/mobile/mobile-app/components/ui/form-control/index.tsx

```
@@ -1,468 +1,468 @@
+	} else if (
+		(props.height !== undefined || props.width !== undefined) &&
+		size === undefined
+	) {
+		return (
+			<UIFormControl.Error.Icon
+				ref={ref}
+				{...props}
+				className={formControlErrorIconStyle({ class: className })}
+			/>
+		)
+	}
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)